openfisca_core/scripts/run_test.py

```diff
@@ -57,8 +57,8 @@ def main():
         'verbose': args.verbose,
         'force': args.force,
         'name_filter': args.name_filter,
-        'default_relative_error_margin': args.relative_error_margin,
```

Comment:
I want the the options to have the same name in the module and [the script](https://github.com/openfisca/openfisca-core/pull/429/files#diff-a058ac150311d34544c2de0cbecfca10R23), and `--default_relative_error_margin` seemed a very long option for a shell script.

@MattiSG do you think it is acceptable ?